google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -123,8 +131,22 @@ public HttpHeaders getHeaders() {
    *
    * @since 1.14
    */
-  public final String getContent() {
-    return content;
+  public final String getContent() throws IOException {
```

Comment:
IOException cannot be throw here.  That's backwards incompatible.

Reply:
Removed